src/routes/docs/tooling/keyboard/+page.markdoc

```diff
@@ -0,0 +1,39 @@
+---
+layout: article
+title: Keyboard
+description: How to use the keyboard with the optimized Appwrite Console for both efficiency and accessibility, and introducing The Appwriter.
```

Comment:
```suggestion
and introducing The Appwriter
```

This is out of context of the page and doesn't match the tone of the docs which should be more serious and functional. This would be something we say in an announcement blog post.